types/react-stripe-elements/package.json

```diff
@@ -7,7 +7,7 @@
     ],
     "dependencies": {
         "@types/react": "*",
-        "@types/stripe-v3": "*"
+        "@types/stripe-v3": "^3.1.33"
```

Comment:
Is the intent here to keep this pointing at DT? If so, this version range isn't really going to help anything; we'll release a shim and the carat will mean that it will point to that new version, which is probably not desired. Any reason to not just point this at the newer v3 types that are being replaced?